### ISSUE TITLE
[RFC] l10n_nl_chart_rgs: upgrade RGS Taxonomie to version 2.0

### ISSUE THREAD

**Issue by astirpe** [OCA/l10n-netherlands#83]
Would it be of interest to upgrade the accounts present in module `l10n_nl_chart_rgs` according to RGS Taxonomie version 2.0?
Version 2.0 was published 12 jan, 2017 ([Download link](http://www.referentiegrootboekschema.nl/kbase/index.php?View=files&CategoryID=1))

**Comment by astirpe** [comment]
The published taxonomie version 2.0 has bugs: some accounts have duplicated codes.
```

WWivGpv 8104000 Geactiveerde productie voor het eigen bedrijf
WWivGpvGpe      8104010 Geactiveerde productie eigen bedrijf
WWivGpvPge      8104020 Privé gebruik eigen bedrijf
WWivWav 8104000 Wijziging agrarische voorraden
WWivWavAaf      8104010 Aanwas fruitopstanden
WWivWavAav      8104020 Aanwas vee
WWivWavOvv      8104030 Overige voorraadmutaties


```

We sent an email to the Support Team RGS, they replied that they are going to fix this issue in a new version.

**Comment by astirpe** [comment]
RGS version 3.0 is available, so this issue became obsolete. Closing.
